src/locales/en.json

```diff
@@ -1,6 +1,7 @@
 {
   "hero": {
     "title": "Search for content to reuse",
+    "subtitle": "Browse over 500 million images, available for reuse",
```

Comment:
This is more of an @annatuma question, but both the title and the subtitle end with "reuse" and I don't think there's any way to actually browse through our collection of images, so "browse" may be confusing.

Reply:
Good points!